### ISSUE TITLE
Create CommunityStakeBanner using CWBanner

### ISSUE THREAD

**Issue by dillchen** [hicommonwealth/commonwealth#5505]
## Description

A banner that displays that the community has enabled stake, with engaging copy that prompts the user to buy

This ticket is specifically about preparing the component, even though the modal it would prompt and the link to the blog post are not yet ready.

DevLink: https://www.figma.com/file/KOEVNGXStBhmX3NnYq7MxP/Community-Staking?type=design&node-id=1117-201428&mode=design&t=B3G4J8YcV2qdXK1R-4

## Project Owner
PM: @dillchen
Designer: @jessmart1213

## Engineering Requirements
- Implement based on the CWBanner

## Acceptance Criteria
- Implement component to Design Specifications, see [devlink](https://www.figma.com/file/KOEVNGXStBhmX3NnYq7MxP/Community-Staking?type=design&node-id=1117-201428&mode=design&t=B3G4J8YcV2qdXK1R-4).
- This banner is from our design system with 1 element that is not specified/breaks the design system: the Caption / regular copy and tag under the CTAs.
- Behavior to implement:
  - As a user, when I click the `x`, the banner should not be shown again for the duration of the user session.
  - As a user, after the first two weeks, and I have not visited since it went live, I should see the banner

- **Not needed for Component Acceptance**, but the following stories will be in a separate (later) ticket:
  - "Buy Stake" Button should open the "Buy Stakes Modal". 
  - As a user, when I click `Learn more`, it should link to the blog post
  - As a user, if a community has `stakes` enabled I should see the modal every visit, unless I exit during  the first two weeks after the community goes live. 
  - After closing the banner, it should re-appears only the latter of, after 2 weeks or the beginning of their next session.

## Additional context

<img width="662" alt="Screen Shot 2023-11-22 at 2 28 31 PM" src="https://github.com/hicommonwealth/commonwealth/assets/5334557/e54819eb-a28f-4e6e-b2e1-e00c496de6ad">

<img width="160" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/5334557/1cd111aa-516a-49e6-b691-5dadcb4bb9a2">

[Mobile](https://www.figma.com/file/KOEVNGXStBhmX3NnYq7MxP/Community-Staking?type=design&node-id=1151-222674&mode=design&t=BVH8vX8wmVfLSNNr-4)

**Comment by masvelio** [comment]
@lzach83 This is most up-to-date figma link https://www.figma.com/file/KOEVNGXStBhmX3NnYq7MxP/Weekly-contests?type=design&node-id=938%3A94050&mode=design&t=LEBVp6mxKpjufwnn-1

**Comment by masvelio** [comment]
blocked by #5560

**Comment by CowMuon** [comment]
Reassigning from ZachLee to Ifu.

**Comment by Miaplacidus** [comment]
@CowMuon @jnaviask 
This is technically blocked by another card, but I'll start in on whatever can be done now.

**Comment by Miaplacidus** [comment]
@dillchen Quick question: has the relevant blog post been created yet? If not, no worries. The component is otherwise complete and behaviors like navigating to the post can be added later. I'm in agreement with @masvelio that behavior around when the banner is displayed should happen in a ticket around implementing the staking feature.
